code/modules/hydroponics/hydroponics.dm

```diff
@@ -775,6 +794,34 @@
 	else if(istype(O, /obj/item/storage/part_replacer))
 		RefreshParts()
 		return
+	else if(istype(O, /obj/item/gun/energy/floragun))

```

Comment:
It's kinda odd that, for this GUN setting, you have to click on a plant tray directly IN MELEE RANGE for it to have any effect.

Also, have you added any effects for if a podperson gets hit by one of these new floral somatoray projectiles yet?

Reply:
Well, it's also odd that we have a gun that shoots plant bullets in the first place. Plus, being able to change a tray's species from a range didn't really feel very natural to do, so close range plant mutation headshots will have to do.

Also, no, not yet, but I also don't want to weaponize this against plant people more than the regular floral somatoray does, so I think I've got a silly effect I can use instead.